cluster-monitoring/README.md

```diff
@@ -47,19 +47,20 @@ This charts only has configurable parameters for opsgenie-heartbeat-proxy, every
 
 The following tables lists the configurable parameters of the prometheus-operator chart and their default values.
 
-Parameter | Description | Default
---- | --- | ---
-`defaultDashboards` | Whether to include the default Grafana dashboards (found in `./dashboards-default`) | `true`
-`extraDashboards` | Extra dashboard files to include (found in `./dashboards-extra`) | `[]`
-`opsgenieHeartbeatProxy.image` | Image | `traumfewo/opsgenie-heartbeat-proxy`
-`opsgenieHeartbeatProxy.imageTag` | Image tag | `v0.0.2`
-`opsgenieHeartbeatProxy.imagePullPolicy` | Image pull policy | `IfNotPresent`
-`opsgenieHeartbeatProxy.port` | Port to run container on | `8080`
-`opsgenieHeartbeatProxy.replicas` | Amount of replicas| `3`
-`opsgenieHeartbeatProxy.imageTag` | Image tag | `v0.0.2`
-`opsgenieHeartbeatProxy.opsgenie_api_key` | Opsgenie API Key | `""`
-`opsgenieHeartbeatProxy.heartbeat_name` | Opsgenie Heartbeat name | `""`
-`opsgenieHeartbeatProxy.resources` | Pod resource requests & limits | `CPU: 128m, Memory: 128Mi`
+| Parameter                                 | Description                                                                                                                                                                                                                                                              | Default                              |
+| ----------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------ |
+| `defaultDashboards`                       | Whether to include the default Grafana dashboards (found in `./dashboards-default`)                                                                                                                                                                                      | `true`                               |
+| `extraDashboards`                         | Extra dashboard files to include (found in `./dashboards-extra`)                                                                                                                                                                                                         | `[]`                                 |
+| `createMonitoringNamespace`               | Whether to create an "application-monitoring" namespace where you can add additional PrometheusRules and/or ServiceMonitors. This namespace will have a `prometheus` label, which can be used with Prometheus' ruleNamespaceSelector and serviceMonitorNamespaceSelector | `false`                              |
```

Comment:
shouldn't this be `true` by default?

Reply:
For us yes, but this chart is public so I figured others might not want it :)